__tests__/run_seed.test.js

```diff
@@ -82,6 +82,92 @@ describe("App testing", () => {
+      return request(app)
+        .get(`/api/articles?sort_by=title`)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles[0].author).toBe("rogersop");
+          expect(body.articles[body.articles.length - 1].author).toBe(
+            "icellusedkars"
+          );
+        });
+    });
+    test("responds with a 404 error when passed a sort_by column query where the passed column does not exist on the database", () => {
+      return request(app)
+        .get(`/api/articles/?sort_by=rainbows`)
+        .expect(404)
+        .then(({ body: { msg } }) => {
+          expect(msg).toBe("Not found!");
+        });
+    });
+    test("Returns a 200 status and accepts a sort-by query sorting by created_at by default", () => {
+      return request(app)
+        .get(`/api/articles`)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles[0].created_at).toBe("2020-11-03T09:12:00.000Z");
+          expect(body.articles[body.articles.length - 1].created_at).toBe(
+            "2020-06-06T09:10:00.000Z"
+          );
+        });
+    });
+    test("200 : accepts an order query, with default set to descending order", () => {
+      return request(app)
+        .get(`/api/articles?sort_by=title&order=ASC`)
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles[0].author).toBe("icellusedkars");
+          expect(articles[articles.length - 1].author).toBe("rogersop");
+        })
+        .then(() => {
+          return request(app)
+            .get(`/api/articles?sort_by=title`)
+            .expect(200)
+            .then(({ body }) => {
+              expect(body.articles[0].author).toBe("rogersop");
+              expect(body.articles[body.articles.length - 1].author).toBe(
+                "icellusedkars"
+              );
+            });
+        });
+    });
+    test("Responds with a 200 status and accepts a topic query", () => {
+      return request(app)
+        .get(`/api/articles?topic=cats`)
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          const catsArticle = {
+            article_id: 5,
+            title: "UNCOVERED: catspiracy to bring down democracy",
+            topic: "cats",
+            author: "rogersop",
+            comment_count: "2",
+            votes: 0,
+            created_at: "2020-08-03T13:14:00.000Z",
+            article_img_url: expect.any(String),
+          };
+          expect(articles[0]).toEqual(catsArticle);
+        });
+    });
+    test("responds with a 404 error when passed a topic that does not exist on the database", () => {
+      return request(app)
+        .get(`/api/articles/?topic=banana`)
+        .expect(404)
+        .then(({ body: { msg } }) => {
+          expect(msg).toBe("Not found!");
+        });
+    });
+    test("400 : responds with error for not accepted order query", () => {
```

Comment:
Please reconsider the order of your tests! You're testing above for the order-by happy path, then the happy and sad paths for the topic query, then into the sad path for order-by - it makes it hard to navigate. (also I love crumpets so NCpoints for that).